pb/lmsysmd/data_ingestion/v1/model.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package lmsysmd.data_ingestion.v1;
+
+import "buf/validate/validate.proto";
+import "google/type/date.proto";
+
+service ModelService {
+  rpc CreateModel(CreateModelRequest) returns (CreateModelResponse) {
+    option idempotency_level = NO_SIDE_EFFECTS; 
+  }
+}
+
+message CreateModelRequest {
+  Id model_id = 1 [ (buf.validate.field).required = true ];
+  Model model = 2 [ (buf.validate.field).required = true ];
+}
+
+message CreateModelResponse {
+  Id model_id = 1 [ (buf.validate.field).required = true ];
+  Model model = 2 [ (buf.validate.field).required = true ];
+}
+
+message Model {
+  Id id = 1 [ 
+    (buf.validate.field).required = true, 
+    (buf.validate.field).enum.defined_only = true 
+  ];
+  string display_name = 2 [ 
+    (buf.validate.field).required = true, 
+    (buf.validate.field).string.min_len = 2 
+  ];
+  Provider provider = 3 [ 
+    (buf.validate.field).required = true, 
+    (buf.validate.field).enum.defined_only = true 
+  ];
+  string model_url = 4 [ 
+    (buf.validate.field).required = true,
+    (buf.validate.field).string.uri = true
+  ];
+  string model_provider_url = 5 [ 
+    (buf.validate.field).required = true,
+    (buf.validate.field).string.uri = true
+  ];
+  google.type.Date release_date = 6 [ (buf.validate.field).required = true ];
+}
+
+// More models will be added in the future.
+enum Id {
+  ID_UNSPECIFIED = 0;
+  GPT_3_5 = 1;
```

Comment:
These ids do not have the version suffix in the api docs

Reply:
Could you please clarify this comment a bit further? Are you referring to the model API docs?

![image](https://github.com/Lev1ty/lmsysmd/assets/56877739/d231120e-f13b-4b40-9f22-1f3a765fa680)

Reply:
like this 
![Uploading CleanShot 2024-06-30 at 12.43.35@2x.png…]()

Reply:
To confirm you're saying it should be `MODEL_ID_GPT_4_TURBO_2024_04_09` instead of just `MODEL_ID_GPT_4`?